sparrow-py/README.md

```diff
@@ -0,0 +1,28 @@
+# Python Library for Kaskada
+
+`sparrow-py` provides a Python library for building and executing Kaskada queries using an embedded Sparrow library.
+It uses [Pyo3][pyo3] to generate Python wrappers for the Rust code, and then provides more Python-friendly implementations on top of that.
+
+[pyo3]: https://github.com/PyO3/pyo3
+
+## Install Python
+
+Use `pyenv` and install at least `3.7` or `3.8`.
+If multiple versions are installed, `nox` will test against each of them.
```

Comment:
Currently the existing python code relies on the CI to test cross versions. I think it would be a good idea to be able to do that locally as well. 

If that is what we want we should add `nox` to the rest of the python code and be consistent.